main.cpp

```diff
@@ -108,9 +108,41 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Foot
+{
+    int stepSize;
+    void stepForward();
 
+};
 
-
+    struct Person
+        {    
+        int age;
+        int height;
+        float hairLength;
+        float GPA;
+        unsigned int SATScore;
+        int distanceTraveled;
+        Foot leftFoot;
+        Foot rightFoot;
+    
+        void run(bool startWithLeftFoot);
+        };
+    
+void Person::run(bool startWithLeftFoot)
```

Comment:
Where is the `howFast` parameter?
- [ ] review the video and instructions, then revise